app/src/main/res/layout/layout_actionbar_home.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All this icons seems a little big in my opinion on real devices. We can check them again with figma and make some changes in the future.